bears/python/PycodestyleBear.py

```diff
@@ -61,3 +67,24 @@ def create_arguments(
         arguments.append(filename)
 
         return arguments
+
+    def process_output(
+            self,
+            output,
+            filename,
+            file,
+            ):
+        output_regex = (r'(?P<line>\d+) (?P<column>\d+) '
+                        r'(?P<message>(?P<origin>\S+).*)')
+        result = re.match(output_regex, output)
+        if not result:
+            return
+        line, column, message, origin = result.groups()
+        yield Result.from_values(
+            origin=self,
+            message=message,
+            file=filename,
+            line=int(line),
+            column=int(column),
+            aspect=LineLength('py'),
```

Comment:
not all pycodestyle errors are LineLength('py').

Please decode the pycodestyle errors and use appropriate aspects